packages/eslint-plugin-template/tests/rules/accessibility-elements-content.test.ts

```diff
@@ -21,7 +21,7 @@ ruleTester.run(RULE_NAME, rule, {
   valid: [
     '<h1>Heading Content!</h1>',
     '<h2><app-content></app-content></h2>',
-    '<h3 [innerHTML]="dangerouslySetHTML"></h3>',
+    '<h3 [innerHtml]="dangerouslySetHTML"></h3>',
```

Comment:
Btw, there's already a test case on line 28 that test the `innerHTML`: 

https://github.com/angular-eslint/angular-eslint/blob/3bc9307b7db838fdf935395ec45a03ca54b58cf9/packages/eslint-plugin-template/tests/rules/accessibility-elements-content.test.ts#L28

Reply:
👍